README.md

```diff
@@ -26,7 +31,24 @@ git pull
 release-it # Follow the instructions
 ```
 
+## Installation
+
+From source using [poetry](https://github.com/python-poetry/poetry).
+
+```
+git clone https://github.com/RedHatQE/rosa-python-client.git
+cd rosa-python-client
+poetry install
+```
+
+From pypi:
+
+```bash
+pip install rosa-python-client --user
```

Comment:
It's better to use `pipx` for this (pipx create venv for each application so your OS stay clean)

Reply:
@myakove when I tried to install it using `pipx install rosa-python-client` getting this, I believe for this package pipx cannot be used
```
No apps associated with package rosa-python-client. Try again with '--include-deps' to include apps of dependent packages, which are listed above. If you
are attempting to install a library, pipx should not be used. Consider using pip or a similar tool instead.
```